PWGLF/Tasks/Resonances/kShortkShort.cxx

```diff
@@ -640,72 +665,73 @@ struct strangeness_tutorial {
 
       ROOT::Math::PxPyPzMVector fourVecDau = ROOT::Math::PxPyPzMVector(daughter1.Px(), daughter1.Py(), daughter1.Pz(), massK0s); // Kshort
 
-      ROOT::Math::PxPyPzMVector fourVecMother = ROOT::Math::PxPyPzMVector(lv3.Px(), lv3.Py(), lv3.Pz(), lv3.M()); // mass of KshortKshort pair
+      ROOT::Math::PxPyPzMVector fourVecMother = ROOT::Math::PxPyPzMVector(lv3.Px(), lv3.Py(), lv3.Pz(), lv3.M()); // mass of LfkShortkShort pair
       ROOT::Math::Boost boost{fourVecMother.BoostToCM()};                                                         // boost mother to center of mass frame
       ROOT::Math::PxPyPzMVector fourVecDauCM = boost(fourVecDau);                                                 // boost the frame of daughter same as mother
       ROOT::Math::XYZVector threeVecDauCM = fourVecDauCM.Vect();                                                  // get the 3 vector of daughter in the frame of mother
 
-      if (TMath::Abs(lv3.Rapidity() < 0.5)) {
+      if (std::abs(lv3.Rapidity() < 0.5)) {
```

Comment:
Bug (reported by the compiler)!

Reply:
Thanks for pointing out the errors.
The error tells "remove the call to 'abs' since unsigned values cannot be negative"
But when I printed the rapidity values it also shows negative values
[68914:lfk-shortk-short]: rapidity is -0.102547
[68914:lfk-shortk-short]: rapidity is 0.364996
[68914:lfk-shortk-short]: rapidity is -0.161002
[68914:lfk-shortk-short]: rapidity is 0.071697
[68914:lfk-shortk-short]: rapidity is 0.331468
[68914:lfk-shortk-short]: rapidity is -0.237249
[68914:lfk-shortk-short]: rapidity is -0.174126
[68914:lfk-shortk-short]: rapidity is -0.0869999

What to do in this case?
P.S. I shifted from TMath::Abs to std::abs on request of o2_linter

Reply:
You have misplaced the closing parentheses of `abs`. You are taking the absolute value of the whole `<` expression, which is `bool`, i.e. `unsigned int`.

Reply:
Okay, I got it.
Thank you.

Reply:
The compiler explicitly tells you that in the error message:
```
error: taking the absolute value of unsigned type 'bool' has no effect
```